stream-video-android-core/src/main/kotlin/io/getstream/video/android/core/call/connection/StreamPeerConnectionFactory.kt

```diff
@@ -164,97 +166,106 @@ public class StreamPeerConnectionFactory(
                 .createInitializationOptions(),
         )
 
+        adm = initAudioDeviceModule()
```

Comment:
Can we make it `internal` ?

Reply:
hmm, actually we can make it private and jsut expose a public method that will update it usage, 
Other classes do not need the adm instance